pkg/bindings/containers/types.go

```diff
@@ -138,6 +138,7 @@ type PruneOptions struct {
 //go:generate go run ../generator/generator.go RemoveOptions
 // RemoveOptions are optional options for removing containers
 type RemoveOptions struct {
+	Depend  *bool
```

Comment:
Also needs changes for the handlers and swagger.

Reply:
Fixed